apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -82,11 +87,27 @@ const TutorialExplorer = React.createClass({
     });
   },
 
-  filterTutorials(filters, hideFilters) {
+  /**
+   * Called when the sort order is changed via dropdown.
+   *
+   * @param {string} value - The new sort order.
+   */
+  handleUserInputSortBy(value) {
+    this.setState({sortBy: value});
+    const { filteredTutorials, filteredTutorialsForLocale } = this.filterTutorialSet(this.state.filters, value);
+    this.setState({
```

Comment:
Any reason we do this as two separate setState calls instead of just one? Doing so will potentially result in two rerenders instead of one.